redbot/cogs/warnings/warnings.py

```diff
@@ -301,19 +346,43 @@ def __init__(self, bot: Red):
 
         await warning_points_add_check(self.config, ctx, user, current_point_count)
         try:
+            dm = await self.config.guild(ctx.guild).toggle_dm()
+            if dm == True:
```

Comment:
You can change it to just:
```suggestion
            if dm:
```